components/dashboard/src/settings/PersonalAccessTokens.tsx

```diff
@@ -398,37 +554,35 @@ function ListAccessTokensView() {
                     </Link>
                 )}
             </div>
+            <>
+                {errorMsg.length > 0 && (
+                    <Alert type="error" className="mb-2">
+                        {errorMsg}
+                    </Alert>
+                )}
+            </>
             <>
                 {tokenInfo && (
                     <>
                         <div className="p-4 mb-4 divide-y rounded-xl bg-gray-50 dark:bg-gray-800">
                             <div className="pb-2">
                                 <div className="flex gap-2 content-center font-semibold text-gray-700 dark:text-gray-200">
-                                    <span className="ml-1">{tokenInfo.data.name}</span>
-                                    <span
-                                        className={
-                                            "font-medium px-1 py-1 rounded-full text-xs" +
-                                            (tokenInfo.method === Method.Create
-                                                ? " text-green-600 bg-green-100"
-                                                : " text-blue-600 bg-blue-100")
-                                        }
+                                    <strong>{tokenInfo.data.name}</strong>
```

Comment:
**issue(non-blocking)**: Using `<strong>` here with the combination of the semibold font weight above makes this text look quite bold. Shall we drop one of these or move the semibold class here?